src/icons/navigator/Home.tsx

```diff
@@ -3,16 +3,9 @@ import Svg, { Path } from 'react-native-svg'
 
 export function Home() {
```

Comment:
The previous version of this file - non Valora branded should be moved to `branding/celo/src/icons`.

Reply:
Yup I will do that